docs/embrace-api/supported_metrics_and_queries.md

```diff
@@ -44,76 +45,62 @@ However, we strongly recommend transitioning to the new metrics to ensure a cons
 | crashes_total_deprecated                     | Number of crashes                                         | app_version, os_version               | five_minute, hourly, daily |
 | sessions_by_device_total_deprecated          | Number of sessions grouped by device model                | app_version, device_model, os_version | hourly, daily              |
 | sessions_total_deprecated                    | Number of sessions                                        | app_version, os_version               | five_minute, hourly, daily |
-| users_deprecated                             | Number of unique users                                    | app_version, os_version               | hourly, daily              |
-
-
-
-### Dimension reduction - "Other"
```

Comment:
This is common across all the metrics, I moved it out of standard metrics section